compliance_controller/wrench_to_joint_vel_pub/config/compliance_settings.yaml

```diff
@@ -0,0 +1,17 @@
+spin_rate:  50 # Hertz
+jacobian_frame_name: base_link  # MoveIt! planning frame. Often the last link in your tf tree
+force_torque_frame_name: base # The frame of incoming force/torque data
+force_torque_topic: wrench # The topic of incoming force/torque data
```

Comment:
Just a user comment here - Should this be able to be modified by a user? I'd love to be able to tune this myself similar to controllers.yaml - perhaps it should even be in controllers.yaml or part of it?

Reply:
Yep, they're intended to be modified by the user! Adding it to controllers.yaml is a good idea.